src/components/TestToolMenu.tsx

```diff
@@ -89,6 +90,16 @@ function TestToolMenu({user = USER_DEFAULT, network}: TestToolMenuProps) {
                     onPress={() => Session.invalidateCredentials()}
                 />
             </TestToolRow>
+
+            <TestToolRow title="Native crash">
```

Comment:
suggestion: hide on web/desktop since we've only implemented force-crash on iOS and Android